templates/upload.html

```diff
@@ -2,6 +2,20 @@
 {% load static %}
 {% block content %}
 
+{% if messages %}
+  {% for message in messages %}
+    <div class="text-center">
+      {% if message.level == DEFAULT_MESSAGE_LEVELS.SUCCESS %}
```

Comment:
If you stick the following into settings.py, we'll get full messages<->Bootstrap compat and then you can then do something like `...other classes... alert-{{ message.tags }}` instead of the if/else casing for the level. (or, since it seems like we aren't using `alert-*` styling b/c of the background color, just `{{ message.tags }}`)
```python
# at the top of the file
from django.contrib.messages import constants as messages

# wherever you want
MESSAGE_TAGS = {
    messages.ERROR: 'danger'
}
```

Reply:
Oh, thanks, I'll look into that later! Good to know--I felt like the if/else casing was a bit unwieldy